UIMentoringSession/MenteesChallenges/Gaming Screen UI Challenge/Maksym Bilyk/MaksymBilykGamingScreenViewController.xib

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Wrong font-weight - you can change it like on screen. font-weight for this label - heavy
<img width="893" alt="Screenshot 2023-05-27 at 09 57 10" src="https://github.com/alexnikol/UIMentoringSessions/assets/14887101/9f0f67d5-34ad-4b5e-beaf-316a157d4e84">